scripts/insert_into_appointments.sql

```diff
@@ -1 +1,13 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date, time, comment, d_family_name, d_given_name, p_family_name, p_given_name) VALUES
+  (10am, '2016-08-12', ' seeing Dr. ', 'Hilton', 'Vicky', 'Wilcox', 'Marsha'),
+  (1pm, '2016-08-17', ' seeing Dr. ', 'Hubbard', 'Spencer', 'Talley', 'Ivory'),
+  (10am, '2016-08-12', ' seeing Dr. ', 'Hubbard', 'Spencer', 'Wilcox', 'Marsha'),
+  (1pm, '2016-08-17', ' seeing Dr. ', 'Hilton', 'Vicky', 'Hodge', 'Blake');
```

Comment:
You were close here but 10am, 1pm, these are not valid inputs for SQL. They should be a time value like '1000' or '1300' and SQL will parse them into times. You could have set it up to take a string instead of a time and then you could have given it '10am' and '1pm'.

Also in your INSERT line you say date then time, but then you pass in time first then date, so watch that.